bin/yaml/dotnet.yaml

```diff
@@ -5,7 +5,8 @@ compilers:
     untar_dir: Core_Root
     check_exe: .dotnet/dotnet --version
     targets:
+      - trunk
```

Comment:
This won't cause trunk to install every night, if that's the intention. You'll need:

```yaml
- name: trunk
  if: nightly
  always_install: true
```

Note that:
1. We don't currently support building trunk images. We would need support in the https://github.com/compiler-explorer/misc-builder/blob/main/build/build-dotnet.sh script to do so.
2. We don't build dot net nightly. That would need changes in https://github.com/compiler-explorer/compiler-workflows to schedule a build every next
3. We would need a configuration in the main CE repo for this, and as far as I can tell there's a dependency in that to the specific version being installed (cc @hez2010 for comment on this, the only person who knows what's going on there!)

Reply:
updated

> We don't currently support building trunk images.

we do have support for trunk: https://github.com/compiler-explorer/misc-builder/blob/3721571a5fcb45f3d6caf167f25832203b8ab5f9/build/build-dotnet.sh#L6

> That would need changes in https://github.com/compiler-explorer/compiler-workflows

i'll update the workflow

Reply:
Well, that's embarrassing! Thanks for taking the time to double check my incorrect statement!! This is great news.